src/languages/en-US.ts

```diff
@@ -1970,6 +1970,12 @@ export default class extends Language {
 		COMMAND_HELP_TITLE: (name, description) => `📃 | ***Help Message*** | __**${name}**__\n${description}\n`,
 		COMMAND_HELP_USAGE: usage => `📝 | ***Command Usage***\n\`${usage}\`\n`,
 		COMMAND_HELP_EXTENDED: extendedHelp => `🔍 | ***Extended Help***\n${extendedHelp}`,
+		COMMAND_SUPPORT_DESCRIPTION: `Show support instructions`,
+		COMMAND_SUPPORT_EXTENDED: builder.display('support', {
+			extendedHelp: `Literally gives you a link to our support server. What did you really expect this to do :thinking:`
```

Comment:
This should have some information about the official server, plus, the message is a little mean, and we do not use `:thinking:` emojis but rather use their unicode.